File: app/src/main/java/com/kittendev/sticker/StickerApplication.java
package com.kittendev.sticker;

import android.app.Application;
import android.os.Environment;

public class StickerApplication extends Application {

    public static final String STICKER_PATH = Environment.getExternalStorageDirectory() + "/Sticker";
    public static final String SERVER_FILE = "http://sticker.kittendev.com/Sticker.zip";
    public static Application instance ;

    @Override
    public void onCreate() {
        super.onCreate();
        instance = this;
    }

}

File: app/src/main/java/com/kittendev/sticker/StickerApplication.kt
package com.kittendev.sticker

import android.app.Application
import android.os.Environment

class StickerApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        instance = this
    }

    companion object {
        val STICKER_PATH = Environment.getExternalStorageDirectory().toString() + "/Sticker"
        val SERVER_FILE = "http://sticker.kittendev.com/Sticker.zip"
        lateinit var instance: StickerApplication
    }

}